Userland/Libraries/LibM/math.cpp

```diff
@@ -703,6 +707,163 @@ long double ceill(long double value) NOEXCEPT
     return internal_to_integer(value, RoundingMode::Up);
 }
 
+#else
+
+double round(double x) NOEXCEPT
+{
+    // Note: This is break-tie-away-from-zero, so not the hw's understaning of
+    //       "nearest", which would be towards even.
+    // based on: https://github.com/gagern/gnulib/blob/6465306025facca3881bcea411b757b0fe8e8df8/lib/round.c#L89
```

Comment:
I believe this would require us to re-license the file under the GPL.

Reply:
What would you recommend?
Would removing that note suffice, or do you think that this algorithm is "unique" to them

Reply:
Looking at the [Wikipedia Article](https://en.wikipedia.org/wiki/Rounding#Round_half_away_from_zero), you could come up with this implementation

Reply:
If we're being strict about this, I think you've basically tainted yourself by looking at a GPL implementation and stating publicly that you've written this implementation based on GPL code.
I think the super clean thing to do here would be to ask someone who hasn't looked at the code to write an implementation based on the Wikipedia text.

Reply:
I asked @kleinesfilmroellchen nicely, so we have a new implementation

We checked, the zero-check is necessary, because parts of our js string formatting does not know about `-0`, and this was easier to do atm. (Number.toPrecission for example)